### PR TITLE
include CRDs during helm templating

### PR DESCRIPTION
**Pull request by paveq**
Fixes https://github.com/crumbhole/argocd-lovely-plugin/issues/63

### CODE REVIEW

**Comment by Joibel** [comment]
Any chance of a basic test in /test to prove it's working. I think I did this once and found that it wasn't working as hoped.

**Comment by paveq** [comment]
> Any chance of a basic test in /test to prove it's working. I think I did this once and found that it wasn't working as hoped.

I added it. Tests however don't run directly on Mac OS, but I think it should work in CI.

File: test/helm_crd/templates/crd.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apiVersion: apiextensions.k8s.io/v1
```
Comment (Joibel) [inline test/helm_crd/templates/crd.yaml:1]: This needs to be in a crds folder, not templates

**Comment by Joibel** [comment]
You should be able to docker build to run the tests. The current test passes without the helmProcessor.go change. The crd needs to be placed in a folder called `test/helm_crd/crds` rather than `test/helm_crd/templates`, and then it's a good test. Perhaps this is what I needed to discover before.
Otherwise LGTM and I'll do a release if you can fix up the test.

**Comment by paveq** [comment]
Yep, indeed you are correct. I've moved the crd now, and verified without the flag to helm it will not be in output.

**Pull request merged**